sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/models/AttestationSignerCollection.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.attestation.models;
+
+import com.azure.core.annotation.Immutable;
+
+import java.util.List;
+
+/**
+ * An AttestationSignerCollection represents a collection of {@link AttestationSigner} objects.
+ */
+@Immutable
+public interface AttestationSignerCollection {
```

Comment:
Was this part of arch review? Having return a type rather than a list? Or was the reason more about we may in future add more things to it and hence have a type.

Reply:
Yes, it was a part of the arch review - and the purpose was to enable future expansion of the type.